### PR TITLE
Aws amicleaner

### PR DESCRIPTION
**Pull request by ct-hbk**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by NM-HBK** [comment]
I confirm that I am willing to be listed as a maintainer of this recipe.

**Comment by as-hbk** [comment]
I confirm I am willing to be a maintainer of this recipe.

File: recipes/aws-amicleaner/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c8a8721c055e59a20e56dd1796ff1228ae5c165341c104267c43e881e3cd8fdf
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```
Comment (marcelotrevisani) [inline recipes/aws-amicleaner/meta.yaml]:
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
  noarch: python
```

File: recipes/aws-amicleaner/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+requirements:
+  host:
+    - pip
+    - python
+    - awscli
+  run:
+    - pip
```
Comment (marcelotrevisani) [inline recipes/aws-amicleaner/meta.yaml]: do you really need ``pip`` at runtime? I think we can delete it, can we?
```suggestion
```

File: recipes/aws-amicleaner/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c8a8721c055e59a20e56dd1796ff1228ae5c165341c104267c43e881e3cd8fdf
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
```
Comment (xhochy) [inline recipes/aws-amicleaner/meta.yaml]: If you have `noarch`, you need to declare the entrypoints of the package here.

**Pull request merged**